network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -263,7 +263,7 @@ <h2 class="h3-heading">{% trans "Company contact info" %}</h3>
       <div class="row">
         <div class="{{section_class}} mt-5 pt-3 bg-white product-section-padding-x">
           {% if product.updates|length > 0 or coralTalkServerUrl %}
-          <h3 class="pni-product-smaller-body-bold mb-4 mb-md-5">{% trans "Updates" %}</h3>
+          <h3 class="h2-heading mb-4 mb-md-5">{% trans "Updates" %}</h3>
```

Comment:
This line was accidentally updated by me in my other PR. It should be using `h2-heading`.